src/Scrutor/TypeSourceSelector.cs

```diff
@@ -154,7 +154,7 @@ private IImplementationTypeSelector InternalFromAssembliesOf(IEnumerable<Type> t
 
     private IImplementationTypeSelector InternalFromAssemblies(IEnumerable<Assembly> assemblies)
     {
-        return AddSelector(assemblies.SelectMany(asm => asm.DefinedTypes).Select(x => x.AsType()));
+        return AddSelector(assemblies.SelectMany(asm => asm.ExportedTypes);
```

Comment:
```suggestion
        return AddSelector(assemblies.SelectMany(asm => asm.ExportedTypes));
```

This should fix the build error!